tox.ini

```diff
@@ -7,4 +7,15 @@ commands = pytest
 
 [testenv:temperature]
 basepython= python3.6
-commands = python main.py
\ No newline at end of file
+commands = python main.py
+
+[docs]
+changedir = docs
+deps =
+    -rdocs/requirements.txt
+
+[testenv:docs]
+changedir = {[docs]changedir}
+deps = {[docs]deps}
+commands =
+    sphinx-build -W -b html source {envtmpdir}/html
```

Comment:
why not make? or integrate this command in make and, again, do make.

Reply:
We can change it. I was building from a template. Will need to do more testing to make sure that changed command works.

Reply:
Well, actually, readthedocs should be doing the make.

Reply:
Hm, need to research this. Does it make sense to test doc gen

Reply:
If we plan on auto publishing docs (which we probably should), we will want to definitely test doc gen.

Reply:
Sounds good. I will try to do the CI later today, but I expect today and tomorrow to be a bit busier at work.

Reply:
No problem, was just trying to get some stuff done to prep this for larger consumption.

Hopefully you don't mind me piggybacking your project for the learning exp.

Reply:
Not at all, I find I learn a lot too, enjoy the work and the project is going good so please keep them coming :)